test/util/utils.ts

```diff
@@ -1,88 +1,70 @@
-import { stringToTerm, termToString } from 'rdf-string';
-import { ExpressionError } from '../../lib/util/Errors';
+import type { AliasMap } from './Aliases';
 import type { GeneralEvaluationConfig } from './generalEvaluation';
-import { generalEvaluate } from './generalEvaluation';
+import type { Notation } from './TruthTable';
+import { UnaryTable, BinaryTable, VariableTable } from './TruthTable';
 
-export function testAll(exprs: string[], config?: GeneralEvaluationConfig) {
-  exprs.forEach(_expr => {
-    const expr = _expr.trim();
-    const matched = expr.match(/ = [^=]*$/ug);
-    if (!matched) {
-      throw new Error(`Could not match '${expr}'`);
-    }
-    const equals = matched.pop();
-    const body = expr.replace(equals, '');
-    const _result = equals.replace(' = ', '');
-    const result = stringToTerm(replacePrefix(_result));
-    // Console.log(`${expr}\n${equals}\n${body}\n${_result}\n${result}`);
-    it(`${body} should evaluate to ${_result}`, async() => {
-      const evaluated = await generalEvaluate({
-        expression: template(body), expectEquality: true, generalEvaluationConfig: config,
-      });
-      expect(termToString(evaluated.asyncResult)).toEqual(termToString(result));
-    });
-  });
+export interface ITestArgumentBase {
```

Comment:
I don't really like the name. What about ITestTableConfigBase?

Reply:
Sure!